plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -412,3 +429,14 @@ func MapPodsToMachines(lister algorithm.PodLister) (map[string][]*api.Pod, error
 	}
 	return machineToPods, nil
 }
+
+func haveSame(a1, a2 []string) bool {
```

Comment:
this func test if two arrays has at least one common element. It returns true once the first one is encountered, and false if no common element is found.